src/autoscroll.ts

```diff
@@ -113,6 +117,13 @@ export class AutoScroll extends Behavior {
     let lastScrollHeight = self.document.scrollingElement.scrollHeight;
 
     while (this.canScrollMore()) {
+      if (document.location.pathname !== this.origPath) {
+        behaviorLog("Location Changed, stopping scroll: " +
+          `${document.location.pathname} != ${this.origPath}`, "info");
+        addLink(document.location.href);
```

Comment:
If the location has already changed by this point, should we grab the URL prior to this?

Reply:
This is called in a loop while page is scrolling, so should be the quickest we detect the change due to scrolling. I suppose we can add a more generic behavior that detects this not tied to scrolling, but for now this should be fine.